Automattic/jetpack
# Contact Form: Toolbar Improvements

Fixes #27029

#### Changes proposed in this Pull Request:

* Include a Toolbar dropdown that allows managing some Forms settings

![image](https://user-images.githubusercontent.com/7811225/198137028-5576762d-4432-417e-b7d2-dfcdd93c009c.png)


#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Open/create a Post/Page and add a `Form` block
* Select the Form block and check if the new button is present in the Toolbar
* Play with the settings to ensure they work as expected
